src/actions/posts.js

```diff
@@ -2,15 +2,15 @@
 // See License.txt for license information.
 
 import {bindClientFunc} from './helpers.js';
-import Client from 'client/client_instance';
-import {PostsTypes as types} from 'constants';
+import Client from 'client';
+import {PostsTypes} from 'constants';
 
 export function fetchPosts(teamId, channelId) {
     return bindClientFunc(
         Client.fetchPosts,
-        types.FETCH_POSTS_REQUEST,
-        types.FETCH_POSTS_SUCCESS,
-        types.FETCH_POSTS_FAILURE,
+        PostsTypes.FETCH_POSTS_REQUEST,
+        PostsTypes.FETCH_POSTS_SUCCESS,
```

Comment:
This has to be changed to use `RECEIVED_POSTS` along with the success so it can be reduces by the requests store and the entities part of the store